src/_P079_Wemos_Motorshield.ino

```diff
@@ -1,48 +1,115 @@
+//   -> https://github.com/wemos/LOLIN_I2C_MOTOR_Library
+//
+// Wemos Motor Shield V1.0 requires updated firmware to prevent i2c bus lockups.
+// See: https://hackaday.io/project/18439-motor-shield-reprogramming
+//
+// Note: see wiki for Wemos V1 setup. motor_shield.bin needs to be flashed first!!!
 // see wiki: https://www.letscontrolit.com/wiki/index.php?title=WemosMotorshield
-
+//
+// Board Docs:
+//   Wemos V1.0 Motor Shield: https://diyprojects.io/test-shield-motor-i2c-wemos-d1-mini-pro-drive-2-motors-15vdc
+//   Lolin V2.0 Motor Shield: https://docs.wemos.cc/en/latest/d1_mini_shiled/motor.html
+//   TB6612FNG DC Motor Driver: https://toshiba.semicon-storage.com/info/docget.jsp?did=10660&prodName=TB6612FNG
+//
+// Command Syntax (WemosMotorShieldCMD and LolinMotorShieldCMD keywords are interchangeable):
+//   WemosMotorShieldCMD,<Motornumber [0,1]>,<Forward/Backward/Stop/Standby/Brake>,<Speed [0-100]>
+//   LolinMotorShieldCMD,<Motornumber [0,1]>,<Forward/Backward/Stop/Standby/Brake>,<Speed [0-100]>
+// Command Examples:
+//   WemosMotorShieldCMD,0,Forward,100
+//   WemosMotorShieldCMD,1,Stop
+//   LolinMotorShieldCMD,0,Reverse,25
+//   LolinMotorShieldCMD,1,Standby
+//   LolinMotorShieldCMD,1,Brake
+// Command Example for i2c Communication test (Lolin V2 board only)
+//   LolinMotorShieldCMD
+//
+// ************************************************************************************************
 
 #define PLUGIN_079
 #define PLUGIN_ID_079         79
-#define PLUGIN_NAME_079       "Motor - Wemos Motorshield"
-#define PLUGIN_VALUENAME1_079 "Wemos Motorshield"
+#define PLUGIN_NAME_079       "Motor - Wemos/Lolin Motorshield"
+#define PLUGIN_VALUENAME1_079 "Motorshield"
+#define PLUGIN_DEF_NAME1_079  "Wemos_DC_Motor"
+#define PLUGIN_DEF_NAME2_079  "Lolin_DC_Motor"
+
+#define DEF_I2C_ADDRESS_079  0x30
+#define I2C_ADDR_PCFG_P079   PCONFIG(0)
+#define SHIELD_VER_PCFG_P079 PCONFIG(1)
+#define MOTOR_FREQ_P079      1000
+#define PRODUCT_ID_I2C_LOLIN 0x02
+
+// Command keywords: The two Command Names are interchangeable.
+#define CMD_NAME_LOLIN "LolinMotorShieldCMD"
+#define CMD_NAME_WEMOS "WemosMotorShieldCMD"
+
+// Compiler Options
+// #define VERBOSE_P079                 // Uncomment to enable Verbose info log status messages.
+
+// ************************************************************************************************
+
+enum BOARD_TYPE
+{
+  WEMOS_P079 = 0x01,
+  LOLIN_P079
+};
+
+enum MOTOR_STATES
+{
+  MOTOR_STOP = 0x00,
+  MOTOR_FWD,
+  MOTOR_REV,
+  MOTOR_STBY,
+  MOTOR_BRAKE
+};
+
+static uint8_t Plugin_079_MotorShield_address = DEF_I2C_ADDRESS_079;
+static uint8_t Plugin_079_MotorShield_type    = WEMOS_P079;
 
 // copied from <WEMOS_Motor.h>
 #ifndef __WEMOS_MOTOR_H
 # define __WEMOS_MOTOR_H
 
-# if ARDUINO >= 100
+# if (ARDUINO >= 100)
  #  include "Arduino.h"
-# else // if ARDUINO >= 100
+# else // if (ARDUINO >= 100)
  #  include "WProgram.h"
-# endif // if ARDUINO >= 100
+# endif  // if (ARDUINO >= 100)
```

Comment:
Is this still needed?
Arduino 1.0.0 is ancient and I doubt it was ever present on ESP82xx.

Reply:
That particular Arduino definition block appears to be legacy code from the original Wemos Motor Shield Library that was integrated into the plugin.  ESPEasy compiles and runs fine without it.  

My opinion: Seems safe to delete.

- Thomas